administrator/components/com_fabrik/models/fields/fabrikeditor.php

```diff
@@ -68,10 +68,9 @@ protected function getInput()
 
 		$mode      = $this->getAttribute('mode', 'html');
 		$theme     = $this->getAttribute('theme', 'github');
-		$height    = $this->getAttribute('height', '200px');
+		$height    = $this->getAttribute('height', '300px');
 		$maxHeight = $this->getAttribute('max-height', str_ireplace('px', '', $height) * 2 . 'px');
-		$width     = $this->getAttribute('width', '300px');
```

Comment:
$width should be set to 100%
		$width     = $this->getAttribute('width', '100%');